app/controllers/staff/activity_forms_controller.rb

```diff
@@ -3,6 +3,8 @@ class Staff::ActivityFormsController < Staff::BaseController
   include DateHelper
   include ActivityHelper
 
+  DEFAULT_PROGRAMME_STATUS_FOR_FUNDS = "07"
```

Comment:
Glad you've given this a helpful name. Is there a table anywhere that would let us refer to `07` status by its name so we know what it means?

Reply:
Yes, the values for each code can be found on this file: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/locales/codelists/2_03/iati.en.yml